daprdocs/content/en/getting-started/quickstarts/serviceinvocation-quickstart.md

```diff
@@ -304,6 +304,7 @@ Run the `order-processor` service alongside a Dapr sidecar.
 dapr run --app-port 7001 --app-id order-processor --app-protocol http --dapr-http-port 3501 -- dotnet run
 ```
 
+Add this code block to the Program.cs file in the current directory
```

Comment:
This is actually not necessary. The quickstarts are self-contained without users needing to create/add anything for them to work. Instead, we can say something like:

```suggestion
Below is the working code block from the order processor's `Program.cs` file.
```

And then update the code block, since it looks like it's been updated since they were first documented:

```csharp
app.MapPost("/orders", (Order order) =>
{
    Console.WriteLine("Order received : " + order);
    return order.ToString();
});
```